elements/pl-rich-text-editor/pl-rich-text-editor.js

```diff
@@ -34,3 +34,45 @@ window.PLRTE = function (uuid, options) {
     );
   });
 };
+
+// Override default implementation of 'formula'
+
+var Embed = Quill.imports.parchment.Embed;
+
+class MathFormula extends Embed {
+  static create(value) {
+    const node = super.create(value);
+    if (typeof value === 'string') {
+      this.waitUntilLoaded(node, value);
+    }
+    return node;
+  }
+
+  static waitUntilLoaded(node, value) {
+    if (document.readyState !== 'complete') {
+      window.setTimeout(this.waitUntilLoaded, 200, node, value);
+    } else {
+      let html = MathJax.tex2chtml(value);
+      let formatted = html.innerHTML;
+      node.innerHTML = '&#65279;' + formatted + '&#65279;' + ' ';
+      MathJax.typeset();
```

Comment:
Why is this call to `typeset` necessary?

Reply:
Typeset is used to "render" the LaTeX, since Mathjax does some weird things with the CSS to render the LaTeX. WIthout it, the LaTeX just doesn't render